lib/modules/platform/github/branch.ts

```diff
@@ -0,0 +1,38 @@
+import { Http, HttpError } from '../../../util/http';
+import { Result } from '../../../util/result';
+
+function headRef(
+  http: Http,
```

Comment:
we probably should move the http instance to common, like other platforms, so we don't need to pass it through. 🤔